Add "Shipment Threshold Value Check" point in Inventory-rules.md and excluded from manage-product-store.md (#1364)
hotwax/oms-documentation: documents/retail-operations/orders/order-routing/inventory-rules.md

* **All items available anywhere:** Only allocate a shipgroup if all the items are available. All items do not have to be available at the same location, only available at the same time. This is useful if an order should only be shipped if all of it can be shipped without cancellation.
* **Turn off the facility order limit check:** Retailers can set[ fulfillment capacity](https://docs.hotwax.co/documents/system-admins/administration/facilities/configure-fulfillment-capacity) in HotWax, allowing them to define the maximum number of orders a facility can fulfill in a day. Disabling the facility order limit filter gives retailers the flexibility to bypass the defined order limit for a facility, which is especially useful during peak times or high-demand periods. For example, if a retailer turns off this limit, orders can continue to be assigned to that facility even after its maximum capacity has been reached.

> * **Shipment Threshold Value Check:** Retailers can configure a minimum shipment value to prevent order splitting when the item value is too low. This helps avoid the cost of shipping low-value items separately, while still maintaining a balance between delivery speed and overall shipping efficiency. For example, if the threshold is set at $25, only items or item groups exceeding this value will be split and shipped from different locations.

Reviewer: ```suggestion
* **Shipment Threshold Value Check:** Retailers can set a minimum shipment value to prevent order splitting for low-value allocations, ensuring efficient shipping. For example, if the threshold is $25, items will only be allocated if their value exceeds $25.
```
